librarian/routes/lang.py

```diff
@@ -12,20 +12,15 @@
 from bottle_utils.ajax import roca_view
 
 from ..core.contrib.templates.renderer import template
+from ..helpers import lang  # NOQA
 
 
 EXPORTS = {
         'routes': {'required_by':
-                   ['librarian_core.contrib.system.routes.routes']}
+                   ['librarian.core.contrib.system.routes.routes']}
 }
```

Comment:
If you guys agree,we can do it now, I tried to limit the amount of changes to only necessary ones to get it running as agreed. but I'm fine with doing this,sure